src/ignitus-StudentSignUp/Components/Profile.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import progress from '../Images/Group12.1.png';
+import image_map from '../Images/image-map.png';
+import { withErrorBoundary } from '../../ignitus-Internals';
+
+import '../Styles/style.scss';
+
+const Signup = () => {
+  
+  return (
+    <div className="col-lg-12" style={{height: "65vw", backgroundColor: "white"}}>
+    <div className="progress-bar-signup" ><img  src={progress}></img></div>
+      <div className="welcome">Welcome, Sophia</div>
+      <div className="content1">Let’s start with your profile. This will get you successful on Ignitus</div>
```

Comment:
You can use **class** `paragraph` instead of `content1`, and please put all text inside `p` instead of `div`.